R/caretList.R

```diff
@@ -255,6 +255,25 @@ is.caretList <- function(object){
   is(object, "caretList")
 }
 
+#' @title Coerce object to caretList object
+#' @description Coerces list of caret models into a caretList
+#' @param list of caret models
+#' @return a \code{\link{caretList}} object
+#' @export
```

Comment:
I finally got it working like it should! Whew that was hard I don't have much experience with this type of work. I followed this template after referencing what was done on the `data.table` project. Will make that changes next time I get. This way is much much better.

```
#' @title Check if an object is a caretList object
#' @description Check if an object is a caretList object
#' @param object an R object
#' @export
is.caretList <- function(object){
  is(object, "caretList")
}

#' @export
as.caretList <- function(object){
  UseMethod(object)
}

#' @export
as.caretList.caretList <- function(obj) {
  cat("This is a generic function\n")
}
```